### ISSUE TITLE
Add a GitHub Action to deploy Docs site on-demand

### ISSUE THREAD

**Issue by sabbyanandan** [hirosystems/docs#366]

**Comment by LakshmiLavanyaKasturi** [comment]
We currently have a re-deploy option available in Vercel for a specific build. 
![Image](https://github.com/hirosystems/docs/assets/55154433/6b5e813f-5e5f-429d-bcef-e285ea7f7d5e)

If we decide to keep the remote docs and automate the process of build trigger, this [new issue](https://github.com/hirosystems/ordinals-api/issues/103) can address it.

**Comment by LakshmiLavanyaKasturi** [comment]
@sabbyanandan, Since we already have the option to redeploy, I believe we don't need an extra action written.

**Comment by sabbyanandan** [comment]
As far as the "redeploy" option pulls all the upstream repo changes for remote-files, yes, let's please close this issue.

**Comment by LakshmiLavanyaKasturi** [comment]
Using the "redeploy button", the vercel build is successful; however, Netlify is pulling the old cache.

Eg: The Chainhook repo has new doc changes merged into the Chainhook repo.
The docs repo pulls those changes using the remote docs plugin using this PR - https://github.com/hirosystems/docs/pull/408. 

The vercel build was successful; however, Netlify failed. Pushing a new commit to the docs branch refreshed the Netlify cache, and the build was successful.

**Comment by LakshmiLavanyaKasturi** [comment]
Depends on #https://github.com/hirosystems/devops/issues/1267

**Comment by LakshmiLavanyaKasturi** [comment]
Netlify is removed from the build process. hence marking this as complete.
